mmv1/products/compute/api.yaml

```diff
@@ -14042,6 +14049,16 @@ objects:
         name: 'diskSizeGb'
         description: 'Size of the snapshot, specified in GB.'
         output: true
+      - !ruby/object:Api::Type::String
+        name: 'chainName'
+        allow_empty_object: true
+        description: |
+          Creates the new snapshot in the snapshot chain labeled with the 
+          specified name. The chain name must be 1-63 characters long and 
+          comply with RFC1035. This is an uncommon option only for advanced 
+          service owners who needs to create separate snapshot chains, for 
+          example, for chargeback tracking.  When you describe your snapshot 
+          resource, this field is visible only if it has a non-empty value.
```

Comment:
`allow_empty_object` field can be removed.. This only [operates on nested fields](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/type.rb#L69-L77). What was your intention in adding this?